specification/eventgrid/data-plane/Microsoft.MachineLearningServices/stable/2018-01-01/MachineLearningServices.json

```diff
@@ -17,7 +17,7 @@
         },
         "modelVersion": {
           "description": "The version of the model that was registered.",
-          "type": "integer"
+          "type": "string"
```

Comment:
This is a breaking change in a swagger marked as stable.  Is this to fix a mismatch or is the service actually changing?

Reply:
It is a service side change as we want to support symver etc. We will publish event with different dataVersion value. Are there anything else we should do?

Reply:
also Az Machine Learning is a public preview event publisher. We are not GA yet.

Reply:
Usually breaking changes are made in a new API version, to avoid breaking customers using the current API version.  Is this not possible?

Reply:
Why is the swagger in a `stable` directory?  This is how we identify preview/stable API versions.